src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/task/Task.java

```diff
@@ -519,5 +522,13 @@ public ZonedDateTime getReconfigureRequestTime() {
     public ZonedDateTime getLastReconfigurationTime() {
         return lastReconfigurationTime;
     }
+
+    public String getTerminationProcess() {
+        return terminationProcess;
+    }
+
+    public void setTerminationProcess(String terminationProcess) {
```

Comment:
Generally we want to avoid setters to keep objects immutable.  So we'd want to remove this when we have switched it on as the only reason it is here is because we need to be able to retrieve the feature flag from LD.